_people/Samuel_J_Dunham.markdown

```diff
@@ -0,0 +1,31 @@
+---
+name: Samuel J. Dunham
+lastname: Dunham
+group: postdocs
+position: Postdoctoral Scholar
+institution: California Institute of Technology (Caltech)
+advisor: Elias Most
+email: dunhamsj@caltech.edu
+website: https://www.samueljdunham.com
+specialties: General relativistic hydrodynamics, numerical methods
+---
+
+##### Research Projects
+
+I am working on developing theoretical models for dissipative and resistive
+general relativistic magnetohydrodynamics inspired by 14-moment methods
+for application to, among other things,
+pulsar magnteospheres and low-luminosity black hole accretion disks.
+
+##### Biography
```

Comment:
Maybe add PhD institution(s) and what you have worked on?

Reply:
I added a description of my PhD work and included the institutions I worked for.